storage/s3/src/main/java/io/aiven/kafka/tieredstorage/storage/s3/S3MultiPartOutputStream.java

```diff
@@ -67,14 +68,23 @@ public class S3MultiPartOutputStream extends OutputStream {
     public S3MultiPartOutputStream(final String bucketName,
                                    final ObjectKey key,
                                    final int partSize,
+                                   final S3Client client){
+        this(bucketName, key, StorageClass.STANDARD, partSize, client);
+    }
+
+    public S3MultiPartOutputStream(final String bucketName,
+                                   final ObjectKey key,
+                                   final StorageClass storageClass,
+                                   final int partSize,
                                    final S3Client client) {
         this.bucketName = bucketName;
         this.key = key;
         this.client = client;
         this.partSize = partSize;
         this.partBuffer = ByteBuffer.allocate(partSize);
         final CreateMultipartUploadRequest initialRequest = CreateMultipartUploadRequest.builder().bucket(bucketName)
-            .key(key.value()).build();
+                .storageClass(storageClass)
+                .key(key.value()).build();
```

Comment:
nit: continuation indent seems to be at 8. Maybe change to 4 on this repo?
```suggestion
            .storageClass(storageClass)
            .key(key.value()).build();
```

Reply:
you are right. I check the original code. it is right style. So it is my bad. let me correct.